mapreduce/task_runner_proxy.py

```diff
@@ -268,19 +297,84 @@ def run_tasks(sql, files_info):
+
+            # mySemaphore = asyncio.Semaphore(3)
+            # tasks = []
+            # await mySemaphore.acquire()
+            logger.info("before map")
+            map_phase = await start_map_phase(
                 is_mapper_in_file=False,
                 mapper=mapper,
-                file_id=file_id,
-            )
-            start_shuffle_phase(file_id=file_id)
-
-            start_reduce_phase(
+                file_id=file_id)
+            # mySemaphore.release()
+            logger.info(str(map_phase))
+            logger.info("after map")
+            # await mySemaphore.acquire()
+            logger.info("before shuffle")
+            shuffle_phase = await start_shuffle_phase(file_id=file_id)
+            logger.info(str(shuffle_phase))
+            logger.info("after shuffle")
+            logger.info("before reduce")
+            reduce_phase = await start_reduce_phase(
                 is_reducer_in_file=False,
                 reducer=reducer,
                 file_id=file_id,
-                source_file=from_file
-            )
+                source_file=from_file)
+            logger.info(str(reduce_phase))
+            logger.info("after reduce")
+
+            # async with ClientSession() as session:
+            #     loop = asyncio.get_event_loop()
+            #     logger.info("before map inside session")
+            #     loop.run_until_complete(start_map_phase(
+            #         session=session,
+            #         is_mapper_in_file=False,
+            #         mapper=mapper,
+            #         file_id=file_id
+            #     ))
+            # asyncio.run(start_map_phase(
+            #     # session=session,
+            #     is_mapper_in_file=False,
+            #     mapper=mapper,
+            #     file_id=file_id
+            # ))
+            # await start_map_phase(
+            #     session=session,
+            #     is_mapper_in_file=False,
+            #     mapper=mapper,
+            #     file_id=file_id
+            # )
+
+            # loop.close()
+
+            # async with ClientSession() as session:
+
+            # asyncio.run(start_shuffle_phase(
+            #     # session=session,
+            #     file_id=file_id
+            # ))
+            # await start_shuffle_phase(session=session, file_id=file_id)
+
+            # logger.info("return after shuffle and before reduce")
+            # return
+
+            # async with ClientSession() as session:
+
+            # asyncio.run(start_reduce_phase(
+            #     # session=session,
+            #     is_reducer_in_file=False,
+            #     reducer=reducer,
+            #     file_id=file_id,
+            #     source_file=from_file
+            # ))
+            # await start_reduce_phase(
+            #     session=session,
+            #     is_reducer_in_file=False,
+            #     reducer=reducer,
+            #     file_id=file_id,
+            #     source_file=from_file
+            # )
+
```

Comment:
Please remove a bunch of logs or set them on DEBUG level. Also, remove commented code